### ISSUE TITLE
[Fix] too much time to retrieve dataset from COVID-19 Data Hub

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#986]
## Summary
It takes too much time to retrieve dataset from COVID-19 Data Hub. About 2 or 3 minutes when WiFi is week.

## Codes

```Python
import covsirphy as cs
loader = cs.DataLoader("input")
jhu_data = loader.jhu()
```

## Environment

- CovsirPhy version: 2.22.2
- Python version: 3.9.8
- Installation: poetry
- System: WSL (Ubuntu)

**Comment by lisphilar** [comment]
Alternative:
Download ZIP files instead of CSV files.
https://covid19datahub.io/news/index.html#new-urls

**Comment by lisphilar** [comment]
Correct download centre, WebAPI
https://covid19datahub.io/articles/data.html

**Comment by lisphilar** [comment]
This was fixed with #1053. `DataLoader(country: str)` downloads datasets of the specified country.
